controllers/drplacementcontrol_controller.go

```diff
@@ -1755,8 +1755,8 @@ func (r *DRPlacementControlReconciler) updateDRPCStatus(
 
 	r.updateResourceCondition(drpc, userPlacement)
 
-	// do not set metrics if DRPC is being deleted
-	if !isBeingDeleted(drpc, userPlacement) {
+	// set metrics if DRPC is not being deleted and if finalizer exists
+	if !isBeingDeleted(drpc, userPlacement) && rmnutil.IsFinalizerExists(drpc, DRPCFinalizer) {
```

Comment:
use [this](https://github.com/RamenDR/ramen/blob/39485b5adc51671842534660e287ce07503542fd/controllers/drplacementcontrol_controller.go#L1157) instead of defining our own exists routine.

Reply:
done